packages/x-date-pickers/src/internals/models/helpers.ts

```diff
@@ -10,6 +10,8 @@ export type ExtendMui<C, Removals extends keyof C = never> = Omit<
 
 export type MakeOptional<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
 
+export type MakeRequired<T, K extends keyof T> = T & Required<Pick<T, K>>;
```

Comment:
nit: not sure if we need to `Omit` `K` first before making it required. It will work like this, but it is more consistent that way.

```suggestion
export type MakeRequired<T, K extends keyof T> = Omit<T, K> & Required<Pick<T, K>>;
```

Reply:
Sure, will apply it for consistency with `MakeOptional`.
They both work and do the same. 👌

Reply:
Yep ... that's why i marked it as nit! Thanks! 🙇🏼